### DISCUSSION TITLE
so-setup command not found after install

### DISCUSSION THREAD

**Discussion by fuomag9** [Security-Onion-Solutions/securityonion#13767] [2.4] [answered]
### Version

2.4.100

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

other (please provide detail below)

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

20

### RAM

16384

### Storage for /

100GB

### Storage for /nsm

100GB

### Network Traffic Collection

tap

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

No, one or more services are failed (please provide detail below)

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

No, there are no additional clues

### Detail

>When you installed, did you choose Import, Eval, Standalone, Distributed, or something else?

This was never an option for me, an anaconda graphical menu opened and I had to configure basic options. After install I have the error in the screenshot below with the so-setup script missing entirely

![image](https://github.com/user-attachments/assets/e50f0d0a-8c96-4012-a028-43e491be90de)

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by dougburks** [comment, marked as answer]
Did you verify the ISO image as shown at https://securityonion.net/download?

**Reply by fuomag9** [reply]
Solved, it actually got corrupted during transfer. Interesting that it was still corrupted enough to change the install script!
